source/ore/routes/list-users.rst

```diff
@@ -0,0 +1,83 @@
+==========
+List Users
+==========
+
+**GET /api/users**
+
+Returns a list of users based on given criteria.
```

Comment:
I don't really follow. The JSON that is returned for users really just has to do with Ore, there is no personal data attached to it or anything.